cli/request.go

```diff
@@ -23,6 +23,9 @@ func readUnitFiles(fs filesystemspec.FileSystem, dir string) (map[string]string,
 		if fileInfo.IsDir() {
 			continue
 		}
+		if !strings.HasPrefix(fileInfo.Name(), dir) {
+			continue
+		}
```

Comment:
We only want to load unit files. Inago expect unit files to be prefixed, thus I added the check. Had an issue with that. Now it is fixed.